### DISCUSSION TITLE
how do catergories work?

### DISCUSSION THREAD

**Discussion by KoosWelling** [leozide/leocad#734] [Q&A] [answered]
Hello,
Thank you for sharing this very nice CAD editor!
We really like it, because it's multiplatform & easy to use.

We are busy using LeoCAD with some other geometries (I'll add show&post).
But how are categories in LeoCAD working?

From the LDraw site, this should work: "0 !CATEGORY Category name"

The category 'wildcards' in LeoCAD don't seem to look for the meta, but only the filename?

**Comment by leozide** [comment, marked as answer]
The categories work on the description, not the LDraw meta. I thought I had a description somewhere for the syntax but I can't find it.

If you go to the Categories Tab in the Preferences you should be able to look at how the current categories work.
